src/components/policy/policy_external/src/policy_helper.cc

```diff
@@ -46,7 +46,8 @@ namespace {
 
 CREATE_LOGGERPTR_GLOBAL(logger_, "Policy")
 
-bool Compare(const StringsValueType& first, const StringsValueType& second) {
+bool IsDifferentStrings(const StringsValueType& first,
```

Comment:
This function accepts 2 string parameters: first, second.
Return `true` if `first != second`, otherwise return `false`;
So it checks if strings are different.

Reply:
It does not, the value returned is `return (strcasecmp(first_str.c_str(), second_str.c_str()) < 0);`, because this is meant as a comparison function for `std::sort`.

Reply:
@jacobkeeler please check these commits 663b4ed, 4617476
These commits are made as a fixup for a corresponding renaming.